docs/partitions.md

```diff
@@ -1,226 +1,59 @@
-# Partitions
+# Partition Healing
 
-In the original implementation of ringpop, if a cluster is split to multiple
-partitions, nodes in each partition declare each other as faulty, and afterward
-will no longer communicate. Ringpop implemented support for merging the
-partitions, which we call `healing`.
+In the original implementation of ringpop, if a cluster is split to multiple partitions, nodes in each partition declare each other as faulty, and afterward will no longer communicate. Ringpop implemented support for merging the partitions, which we call `healing`.
```

Comment:
If a cluster's **network** is split
Is more appropriate to offer distinction between network and a ringpop partition (with healed network)